rhel7/CMakeLists.txt

```diff
@@ -14,12 +14,18 @@ ssg_build_html_table_by_ref(${PRODUCT} "nist")
 ssg_build_html_table_by_ref(${PRODUCT} "cis")
 ssg_build_html_table_by_ref(${PRODUCT} "cui")
 ssg_build_html_table_by_ref(${PRODUCT} "pcidss")
+ssg_build_html_table_by_ref(${PRODUCT} "anssi")
 
 ssg_build_html_nistrefs_table(${PRODUCT} "common")
 ssg_build_html_nistrefs_table(${PRODUCT} "ospp-${PRODUCT}")
 ssg_build_html_nistrefs_table(${PRODUCT} "C2S")
 ssg_build_html_nistrefs_table(${PRODUCT} "stig-${PRODUCT}-disa")
 
+ssg_build_html_anssirefs_table(${PRODUCT} "anssi_nt28_minimal")
+ssg_build_html_anssirefs_table(${PRODUCT} "anssi_nt28_intermediary")
+ssg_build_html_anssirefs_table(${PRODUCT} "anssi_nt28_enhanced")
+ssg_build_html_anssirefs_table(${PRODUCT} "anssi_nt28_high")
```

Comment:
since the anssi_nt28 is prefix is in evrey single one of them, wouldn't it make sense to prepend it inside the macro?

Reply:
We can prepend it inside the macro. But can we assume that ANSSI Profiles for other products will also have 'anssi' prefix?

Reply:
I agree with both of you :) `anssi` can be moved into macro, but I would leave `nt28` here. Assumption no other profile will be implemented is too strong to me.

Reply:
I have included `anssi_` inside macro. This should have make name of ANSSI tables less redundant, updated in PR description.